src/main/protobuf/v1/tenant.proto

```diff
@@ -49,23 +49,16 @@ message CertifierV1 {
   required string certifierId = 1;
 }
 
-enum PersistentVerificationRenewalV1 {
-  REVOKE_ON_EXPIRATION = 1;
-  AUTOMATIC_RENEWAL    = 2;
-}
-
 message TenantVerifierV1 {
   required string id = 1;
```

Comment:
TIL `reserved` keyword.
It breaks compilation if that field will be mistakenly used in the future.

```suggestion
  reserved 3;
  required string id = 1;
```